frontend/src/pages/components/htmlRenderer.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+
+const HTMLRenderer = ({html}) => {
+    const sanitizedHTML = html.replace(/onerror=/, '').replace(/<a .*onmouseover=.*>/, '<a>');
```

Comment:
Is this enough sanitization? Did you look into something like this? https://www.npmjs.com/package/sanitize-html

Reply:
So ```dangerouslySetInnerHTML``` actually does some sanitization on its own, filtering out <script> tags and the like. I verified about half of [this list](https://owasp.org/www-community/xss-filter-evasion-cheatsheet) to make sure they didn't work, and there were only two types of attacks that worked (the two I added sanitization for).

It's probably not "enough", but the thing is that the customers want us to support hyperlinks, so it's never really going to be totally safe. The href/src validation that that library provides might be worth using though.

I'm not totally sure at what point we'd be going overkill on trying to prevent XSS attacks, when the bigger issue might be just making sure authentication works.

Reply:
Gotcha makes sense!